tools/picard.js

```diff
@@ -299,6 +299,8 @@ function build_ui () {
         default_width: RESOLUTIONS[2][0],
         default_height: RESOLUTIONS[2][1],
     });
+    // Add the .WindowSimple class so we get styling on the scrollbars
+    widgets.window.get_style_context().add_class('WindowSimple');
```

Comment:
Not sure if there is a better way of doing this, but I do think we should keep this specific to e-k-l, and not cause undesired changes on other apps.